testsuite/features/step_definitions/command_steps.rb

```diff
@@ -1566,6 +1569,9 @@
 
   # Cleanup the temporary entry in /etc/hosts on the controller
   `sed -i \'$d\' /etc/hosts`
+
+  hostname = get_target('server').hostname
+  raise 'Wrong server hostname after changing it back' if hostname != new_hostname
```

Comment:
```suggestion
  raise 'Wrong server hostname after changing it back' unless get_target('server').hostname == new_hostname
```

Reply:
I need the assignment to differentiate between the output of the command and the return value